pkg/volume/vsphere_volume/vsphere_volume_util.go

```diff
@@ -104,7 +113,7 @@ func (util *VsphereDiskUtil) CreateVolume(v *vsphereVolumeProvisioner, selectedN
 	// so if we subtract the kubelet path prefix from 256, we are left with 191 characters.
 	// Since datastore name is typically part of volumeName we are choosing a shorter length of 90
 	// and leaving room of certain characters being escaped etc.
-	name := volumeutil.GenerateVolumeName(v.options.ClusterName, v.options.PVName, 90)
+	name := volumeutil.GenerateVolumeName(v.options.ClusterName, v.options.PVName, maxVolumeLength)
```

Comment:
numbers in the comment above does not correspond to the actual code now.

Reply:
I removed this comment altogether.